README.md

```diff
@@ -21,6 +21,49 @@ Currently logs which exceed 65536 characters in length, including any patterns a
 
 -------
 
+Configure Java Util Logging with multiple handlers
+==================
+This library allows you to set up different loggers (java.util.logging.Logger) each of them with a different 
+Logentries handler configuration (e.g. to support different tokens). 
+
+1. Add to logging.properties the following props:
+    - Log configuration class:
+    ```
+    config=com.rapid7.jul.MultipleLogentriesHandlerConfig
+    ```
+
+    - List all the handler names you need:
+    ```
+    logentries.handler.names=logger1,logger2
+    ```
+
+    - Configure each handler using the handler name specified before as prefix:
+
+    ```
+    logger1.com.rapid7.jul.LogentriesHandler.token=4ff1cb0a-beea-4616-b647-1c113de8e7bb
```

Comment:
Just to check again - not real tokens? 😄

Reply:
nope, I used a on line UUID generator :-)